src/GraphQL/Types/Scalars/EnumerationGraphType.cs

```diff
@@ -97,10 +97,14 @@ public void AddValue(EnumValueDefinition value)
         /// <inheritdoc/>
         public override object? Serialize(object? value)
         {
-            if (value == null) // TODO: why? null as internal value may be mapped to some external enumeration name
+            var foundByValue = Values.FindByValue(value);
+
+            // NOTE: for backward compatibility - keep or remove?
+            // See public void allow_null(Type graphType)
+            // g.Serialize(null).ShouldBeNull();
+            if (foundByValue == null && value == null)
                 return null;
```

Comment:
Pretty sure this is required for any field resolver that returns a nullable enum value.

Reply:
OK